<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ListWidgetEditor</class>
 <widget class="QDialog" name="ListWidgetEditor" >
  <property name="objectName" >
   <string notr="true" >ListWidgetEditor</string>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="objectName" >
    <string notr="true" />
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="objectName" >
      <string notr="true" />
     </property>
     <property name="margin" >
      <number>1</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="objectName" >
        <string notr="true" />
       </property>
       <property name="margin" >
        <number>1</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <widget class="QPushButton" name="newItemButton" >
         <property name="objectName" >
          <string notr="true" >newItemButton</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>151</x>
           <y>1</y>
           <width>75</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text" >
          <string>&amp;New Item</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QPushButton" name="moveItemUpButton" >
         <property name="objectName" >
          <string notr="true" >moveItemUpButton</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>151</x>
           <y>130</y>
           <width>75</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <spacer>
         <property name="objectName" >
          <string notr="true" >verticalSpacer</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>151</x>
           <y>65</y>
           <width>75</width>
           <height>59</height>
          </rect>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1" >
        <widget class="QPushButton" name="moveItemDownButton" >
         <property name="objectName" >
          <string notr="true" >moveItemDownButton</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>151</x>
           <y>162</y>
           <width>75</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="deleteItemButton" >
         <property name="objectName" >
          <string notr="true" >deleteItemButton</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>151</x>
           <y>33</y>
           <width>75</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text" >
          <string>&amp;Delete Item</string>
         </property>
        </widget>
       </item>
       <item rowspan="5" row="0" column="0" >
        <widget class="QListWidget" name="listWidget" >
         <property name="objectName" >
          <string notr="true" >listWidget</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>1</x>
           <y>1</y>
           <width>144</width>
           <height>187</height>
          </rect>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="objectName" >
        <string notr="true" >groupBox</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>234</x>
         <y>1</y>
         <width>147</width>
         <height>189</height>
        </rect>
       </property>
       <property name="title" >
        <string>&amp;Item Properties</string>
       </property>
       <layout class="QGridLayout" >
        <property name="objectName" >
         <string notr="true" />
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="2" column="1" >
         <spacer>
          <property name="objectName" >
           <string notr="true" >verticalSpacer_2</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>61</x>
            <y>77</y>
            <width>20</width>
            <height>101</height>
           </rect>
          </property>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="label_2_2" >
          <property name="objectName" >
           <string notr="true" >label_2_2</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>61</x>
            <y>50</y>
            <width>20</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="objectName" >
           <string notr="true" >label_2</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>11</x>
            <y>50</y>
            <width>44</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text" >
           <string>&amp;Pixmap:</string>
          </property>
          <property name="buddy" >
           <cstring>browseButton</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QToolButton" name="previewButton" >
          <property name="objectName" >
           <string notr="true" >previewButton</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>87</x>
            <y>50</y>
            <width>22</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QToolButton" name="browseButton" >
          <property name="objectName" >
           <string notr="true" >browseButton</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>115</x>
            <y>50</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="objectName" >
           <string notr="true" >label</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>11</x>
            <y>21</y>
            <width>44</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text" >
           <string>&amp;Text:</string>
          </property>
          <property name="buddy" >
           <cstring>itemTextLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3" >
         <widget class="QLineEdit" name="itemTextLineEdit" >
          <property name="objectName" >
           <string notr="true" >itemTextLineEdit</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>61</x>
            <y>21</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="objectName" >
      <string notr="true" >verticalSpacer_3</string>
     </property>
     <property name="geometry" >
      <rect>
       <x>9</x>
       <y>206</y>
       <width>382</width>
       <height>40</height>
      </rect>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="horizontalLine" >
     <property name="objectName" >
      <string notr="true" >horizontalLine</string>
     </property>
     <property name="geometry" >
      <rect>
       <x>9</x>
       <y>252</y>
       <width>382</width>
       <height>3</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="objectName" >
      <string notr="true" />
     </property>
     <property name="margin" >
      <number>1</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="objectName" >
        <string notr="true" >helpButton</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>1</x>
         <y>2</y>
         <width>75</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text" >
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="objectName" >
        <string notr="true" >horizontalSpacer</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>82</x>
         <y>1</y>
         <width>54</width>
         <height>28</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="objectName" >
        <string notr="true" >applyButton</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>142</x>
         <y>2</y>
         <width>75</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="objectName" >
        <string notr="true" >okButton</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>223</x>
         <y>1</y>
         <width>77</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="objectName" >
        <string notr="true" >cancelButton</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>306</x>
         <y>2</y>
         <width>75</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ListWidgetEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>269</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel" >
     <x>274</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ListWidgetEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>351</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel" >
     <x>352</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
